tests/integration-tests/tests/update/test_update.py

```diff
@@ -488,7 +472,7 @@ def _check_update_compute(
     cluster.config_file = str(updated_config_file)
     cluster.update()
 
-    logging.info("Sleeping for 180 seconds in case instance type properties cache is not updated yet in jobwatcher")
+    logging.info("Sleeping for 180 seconds in case instance type properties cache is not updated yet in compute config")
```

Comment:
I think we can remove this and the sleep

Reply:
Done